study.md

```diff
@@ -43,21 +49,32 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- the view is which template of the page is being displayed and there can be many view states. 
+
+I still have only some semblance of what view-state is. view state is a magical place where
+all of these manipulations to the front and back end interections of the page/what the user sees
+can be changed without reloading the page? So it's all happening client-side without making continues requests to the server? -->
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+<!-- You can load the information from the server intitially, and then the other code you've written
+can manipulate that one html document to change the view state? a traditional application is instead making
+constant server requests and receiving responses in order to present new information to the user, whereas
+a SPA basically allows the user to manipualte the DOM through your code and layer different information and states into a single page? -->
 ```
 
 ## List some technologies used to make SPA's
 
 What are some of the tools used to create single page applications?
 
 ```md
-<!-- your answer here -->
+<!--
+
+jquery, ajax, html, javascript, MVC , MVVM
+
```

Comment:
These are definitely tools, largely used by the front-end frameworks that actually create the SPA. Ember, React, Angular, Meteor, etc are just a few examples.